resources/uk/gov/hmcts/npm/check-deprecated-npm-dependency.sh

```diff
@@ -6,17 +6,13 @@ REQUIRED_VERSION=${2}
 
 # Attempt to find angular version in a package.json file and trims version of angular/core to get the major version value 
 CURRENT_VERSION=""
-for file in $(find . -type f -name "package.json" -not -path "./node_modules/*"); do
-    version=$(cat "$file" |  grep "$DEPENDENCY" | sed 's/.*"\(.*\)".*/\1/' | tr -d '^' | cut -d '.' -f 1 )
-    if [[ -n "$version" ]]; then
-        CURRENT_VERSION="$version"
-        echo "Current version: $CURRENT_VERSION"
-        break
+version=$(yarn info "@angular/core" --json | jq '.children.Version' | tr -d '"' | cut -d '.' -f 1 )
```

Comment:
```suggestion
version=$(yarn info "@angular/core" --json | jq -r '.children.Version')
```

Reply:
Ah nice thanks -- I still need cut because deprecation map is only looking at the major version https://github.com/hmcts/cnp-jenkins-config/blob/master/deprecation-config.yml#L35